data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -166,7 +164,8 @@
     (validate_missing_participants.delete_records_for_non_matching_participants,
     ),
     (CleanMappingExtTables,),
-    (TemporalConsistency,)
+    (TemporalConsistency,),
+    (NegativeAges,)
```

Comment:
I understand we need to make `NegativeAges` run only during the combined stage- is there a reason to execute it at the end of the stage as opposed to after `clean_years`?

Reply:
No reason, I just added it to the bottom of the list, let me know if there are certain rules that need to be before/after?

Reply:
Not sure about this particular case- all I know is it's safer to preserve the existing order. :man_shrugging: Let me know if you determine that a reordering is needed here.

Reply:
Yep makes total sense, I've moved it back up to where it originally was, thanks for that suggestion!